project/plugins.sbt

```diff
@@ -12,4 +12,4 @@ addSbtPlugin("org.scalameta"        % "sbt-scalafmt"                  % "2.3.0")
 addSbtPlugin("org.scalameta"        % "sbt-scalafmt"                  % "2.4.0")
 addSbtPlugin("org.scoverage"        % "sbt-scoverage"                 % "1.6.1")
 addSbtPlugin("org.scalameta"        % "sbt-mdoc"                      % "2.2.3")
-
+addSbtPlugin("ch.epfl.scala"        % "sbt-bloop"                     % "1.4.3")
```

Comment:
```suggestion
```

Please add this to your [global settings](https://www.scala-sbt.org/1.x/docs/Global-Settings.html#Global+Settings+using+a+Global+Plugin) rather than here. You can probably even retrieve the version dynamically by shelling out to `bloop --help` and parsing the first line, so that it always matches the version you have installed.

Reply:
Not intentional - sorry. Slipped in whilst (lazily) rebasing!